training/distributed_training/pytorch/model_parallel/gpt-neox/smp-train-gpt-neox-sharded-data-parallel.ipynb

```diff
@@ -0,0 +1,924 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #41.        checkpoint_s3_uri=checkpoint_s3_uri if not use_fsx else None,
</pre><p>Shouldn't this be</p><pre class="ql-syntax" spellcheck="false">checkpoint_s3_uri=checkpoint_s3_uri if not use_fsx else hyperparameters["checkpoint-dir"],
</pre><p>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3991/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='50'/>